apps/web/src/components/terminal/PreLaunchSection.tsx

```
@@ -0,0 +1,68 @@
+          className={clsx(
+            'w-full flex items-center justify-center gap-2 py-2 rounded-lg',
+            'border border-dashed border-border',
+            'bg-card/30 hover:bg-card/50',
+            'text-muted-foreground hover:text-foreground-secondary',
+            'transition-all duration-150',
+            'hover:border-primary/50',
+            'group'
+          )}
+          aria-label="Add session"
+          title="Press N to add session"
+        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)